src/applications/personalization/account/tests/containers/AccountApp.redirect.unit.spec.jsx

```diff
@@ -14,7 +14,7 @@ import AccountApp from '../../containers/AccountApp';
 describe('<AccountApp>', () => {
   let wrapper;
 
-  function setUp(featureFlag = true) {
```

Comment:
...and if we delete `AccountApp.jsx` we can kill this file as well.

Reply:
True! Killed in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/14525/files)